Library/Homebrew/os/linux/elf.rb

```diff
@@ -89,13 +96,16 @@ def with_interpreter?
   def dynamic_elf?
     return @dynamic_elf if defined? @dynamic_elf
 
-    @dynamic_elf = if which "readelf"
-      Utils.popen_read("readelf", "-l", to_path).include?(" DYNAMIC ")
-    elsif which "file"
-      !Utils.popen_read("file", "-L", "-b", to_path)[/dynamic|shared/].nil?
-    else
-      raise "Please install either readelf (from binutils) or file."
-    end
+    @dynamic_elf =
+      if HOMEBREW_PATCHELF_RB
```

Comment:
```suggestion
    @dynamic_elf = if HOMEBREW_PATCHELF_RB
```

and unindent all lines below by two spaces

Reply:
@MikeMcQuaid unrelated note, is there a formatter setting for this?, 
for all the above changes i manually forced the indentation. the formatter auto indents it far right

Reply:
If you put the `if` on the next line it'll format like how you've done it and on the same line like I have, I believe.

Reply:
that's what i did before, it leaves 2 extra spaces